thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/tools/Hibernate2JDBCMigration.java

```diff
@@ -27,6 +29,32 @@ public void run() throws Exception {
     initTarget();
     cloneWebappConfig();
     cloneAnomalyFunctions();
+    cloneEmailConfigs();
+  }
+
+  private void cloneEmailConfigs() {
+    EmailConfigurationManager hibernateManager = PersistenceUtil
+        .getInstance(com.linkedin.thirdeye.datalayer.bao.hibernate.EmailConfigurationManagerImpl.class);
+    EmailConfigurationManager jdbcManager = DaoProviderUtil
+        .getInstance(com.linkedin.thirdeye.datalayer.bao.jdbc.EmailConfigurationManagerImpl.class);
+    List<EmailConfigurationDTO> dtoSrcList = hibernateManager.findAll();
+    List<EmailConfigurationDTO> dtoTgtList = jdbcManager.findAll();
+    Set<String> uniques = new HashSet<>();
+    //ALTER TABLE `webapp_config_index` ADD UNIQUE `unique_index`(`name`, `collection`, `type`);
+    for (EmailConfigurationDTO dto : dtoTgtList) {
+      String key = dto.getCollection() + "_" + dto.getMetric();
+      uniques.add(key);
+    }
+    for (EmailConfigurationDTO dto : dtoSrcList) {
+      String key = dto.getCollection() + "_" + dto.getMetric();
+      if (!uniques.contains(key)) {
+        dto.setId(null);
+        jdbcManager.save(dto);
```

Comment:
email config references the function ids as well. We need to compute the mapping with new_function_id and set emailConfigurationDTO.setFunctionIds before calling save